uber/site_sections/utility.py

```diff
@@ -0,0 +1,63 @@
+from uber.common import *
+import subprocess
+import shlex
+
+# admin utilities.  should not be used during normal ubersystem operations except by developers / sysadmins
+
+
+# quick n dirty. don't use for anything real.
```

Comment:
Admittedly, it's not like we're (currently) passing user input here, but a lot of this code is very specific to our deployment, e.g. the assumption that you're running out of a ``git`` checkout.